app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CommentsController < ApplicationController
+  def new
+    @post = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
+    comment = Comment.new(params.require(:comment).permit(:text))
```

Comment:
The comment_params method returns this `params.require(:comment).permit(:text)`, so you should use it here like so 👇🏾 

```Ruby
Comment.new(comment_params)
```